keyboards/lets_split/keymaps/bbaserdem/README.md

```diff
@@ -1,27 +1,70 @@
-# Lets Split Layout
+# Lets Split
 
 Check out [user readme](../../../../users/bbaserdem/README.md) for more info.
+This is still a WIP, but the firmware should be fine.
 
-# Usage
+Builds fine as of 2019-12-17
 
-**These commands depend on there being no other arduino connected!**
-Also udev rules can be set instead of using sudo.
-Please unplug all other usb devices.
+# Setup
+
+I am planning on I2C, using Proton C's (or ProMicro's).
+
+# Caterina
+
+The default bootloader for ProMicro.
+
+For flashing the EEPROM with the side information, flash the following.
+```
+avrdude -p atmega32u4 -P "$(ls /dev/ttyACM*)" -c avr109 -U eeprom:w:"./quantum/split_common/eeprom-lefthand.eep"
+avrdude -p atmega32u4 -P "$(ls /dev/ttyACM*)" -c avr109 -U eeprom:w:"./quantum/split_common/eeprom-righthand.eep"
```

Comment:
This can be replaced with the `:avrdude-split-left` and `:avrdude-split-right` targets.